cleverhans/attacks.py

```diff
@@ -2264,3 +2291,34 @@ def attack(self, x, true_y):
   def attack_class(self, x, target_y):
     adv = self.base_attacker.generate(x, y_target=target_y, **self.params)
     return adv
+
+def arg_type(arg_names, kwargs):
+  """
+  Returns a hashable summary of the types of arg_names within kwargs.
```

Comment:
Please add more details description of the function.
In particular, please add description of arguments and return value.
It would be useful to know that all values of kwargs has to be numpy arrays or TF tensors

Reply:
Added, but let me know what you think of the new version